zuul.d/nodeset.yaml

```
@@ -222,6 +222,26 @@
         nodes:
           - crc
 
+- nodeset:
+    name: centos-9-xxl-2x-centos-9-crc-extracted-2-39-0-xxl
+    nodes:
+      - name: controller
+        label: cloud-centos-9-stream-tripleo-xxl
+      - name: compute-0
+        label: cloud-centos-9-stream-tripleo
+      - name: compute-1
+        label: cloud-centos-9-stream-tripleo
+      - name: crc
+        label: coreos-crc-extracted-2-39-0-xxl
```

We need to change the compute label
```suggestion
      - name: controller
        label: cloud-centos-9-stream-tripleo
      - name: compute-0
        label: cloud-centos-9-stream-tripleo-xxl
      - name: compute-1
        label: cloud-centos-9-stream-tripleo-xxl
      - name: crc
        label: coreos-crc-extracted-2-39-0-xxl
```